client/cache/error_code.go

```diff
@@ -123,6 +123,14 @@ var InvalidSsdbCacheValue = berror.DefineCode(4002022, moduleName, "InvalidSsdbC
 SSDB cache only accept string value. Please check your input.
 `)
 
+var InvalidInitParameters = berror.DefineCode(4002024, moduleName, "InvalidInitParameters", `
+Invalid init cache parameters.
```

Comment:
Invalid init cache parameters.
You can check the related function to confirm that if you pass correct parameters or configure to initiate a Cache instance.

Reply:
@hookokoko please change this

Reply:
All right, I know what do you mean

Reply:
Do you mean that there is no need to define error codes？